spec/api/v1/requests/profile_pic_request_spec.rb

```diff
@@ -0,0 +1,209 @@
+require 'rails_helper'
```

Comment:
nice, I see you added the tests to a new spec for profile pics

Reply:
yeah I switched the previous image_request to profile_pic since it made more sense to make the new ones under that name